modules/promotions/assets/js/react/app.js

```diff
@@ -0,0 +1,35 @@
+import ReactUtils from 'elementor-utils/react';
+import { DirectionProvider, LocalizationProvider, ThemeProvider } from '@elementor/ui';
+import PropTypes from 'prop-types';
+// import HomeScreen from './components/home-screen';
+
+const App = ( props ) => {
+	return (
+		<DirectionProvider rtl={ props.isRTL }>
+			<LocalizationProvider>
+				<ThemeProvider colorScheme={ props.colorScheme }>
+					Hello world
+				</ThemeProvider>
+			</LocalizationProvider>
+		</DirectionProvider>
+	);
+};
+
+const isRTL = elementorCommon.config.isRTL,
+	rootElement = document.querySelector( '#wpcontent' );
```

Comment:
We will need to handle this inside the editor control panel and add the correct selector.